dts/bindings/i2c/zephyr,sim-i2c.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: Native posix I2C emulator
```

Comment:
Maybe also update the description to remove the reference to native_posix

Also please in this commit ("drivers: i2c: Implement the slave interface for posix") , change the commit title and description

Reply:
done